spec/amp-actions-and-events.md

```diff
@@ -71,84 +71,123 @@ For example, the following is possible in AMP.
 ```
 
 ## Element Specific Events
+### form
```

Comment:
@bpaduch for review. I changed the layout of this page (now each component gets its own table rather than all in one table) also added actions and their description for amp-video, amp-youtube.